src/app/components/dynamic-hub/dynamic-hub.component.ts

```diff
@@ -45,7 +45,11 @@ export class DynamicHubComponent implements OnInit {
         this.permissionService.get('persist/resource/config.json').subscribe((resources: Resource[]) => {
             this.canCreateDashboard = (resources.filter(r => r.verb === 'POST').length > 0);
         });
-        this.fetchCards();
+        if (!!this.arlasSettingsService.getSettings().authentication
```

Comment:
what happens if there is no authentication at all ? we will never fetch cards no ?

Reply:
yes if force_connect=false or if authentication is not defined at all, we need test the both

Reply:
Done